test/e2e/upgrade_clusterctl_test.go

```diff
@@ -197,6 +199,21 @@ func preInitFunc(clusterProxy framework.ClusterProxy) {
 		Expect(clusterProxy.Apply(ctx, certManagerYaml)).ShouldNot(HaveOccurred())
 	}
 
+	By("Fetch manifest for bootstrap cluster")
+	path := filepath.Join(os.Getenv("CAPM3PATH"), "scripts")
+	cmd := exec.Command("./fetch_manifests.sh") // #nosec G204:gosec
```

Comment:
is there a way to avoid this invoking of shell inside go code? As you can see you have to ignore the gosec error and we are trying to get rid of such invocations from other places too.

Reply:
Running of these script are part of e2e test and used  the approach in different place of e2e tests. I think we need to check for a  solution for that separately with new ticket.

Reply:
theres already ticket for that , I am concerned about adding new cases when old cases are being worked upon.